selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/chat/ChatLocator.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Please format the code with eXo Formatter Settings http://developer.exoplatform.org/#id-formatter-settings